src/LintingViewExtension/LinterViewModel.cs

```diff
@@ -18,14 +19,12 @@ namespace Dynamo.LintingViewExtension
 {
     public class LinterViewModel : NotificationObject
     {
-        private const string NONE_DESCRIPTOR_GUID = "7b75fb44-43fd-4631-a878-29f4d5d8399a";
-
         #region Private Fields
         private static Uri linterViewHelpLink = new Uri("LintingViewExtension;LinterViewHelpDoc.html", UriKind.Relative);
         private LinterExtensionDescriptor activeLinter;
```

Comment:
I think we can remove this member and rely solely on LinterManager for the ActiveLinter property